src/Sylius/Component/Product/Model/Product.php

```diff
@@ -99,6 +104,7 @@ public function __construct()
         $this->variants = new ArrayCollection();
         $this->options = new ArrayCollection();
         $this->createdAt = new \DateTime();
+        $this->enabled = false;
```

Comment:
I'm :-1: on this, IMO it should be enabled by default.
Otherwise it'll require an extra unnecessary step.

I mean, there's no use case I can think of when you create a product you do not want straight away.
If you do, it means that it should be release at a later date, so you'll use "release on" I guess.